modules/build/src/main/scala/scala/build/internal/NativeBuilderHelper.scala

```diff
@@ -0,0 +1,65 @@
+package scala.build.internal
+
+import java.math.BigInteger
+import java.security.MessageDigest
+
+import scala.build.Build
+import scala.scalanative.{build => sn}
+
+case object NativeBuilderHelper {
```

Comment:
NIT: no need to make that a `case` (`case object` is more for object that represent some data).

```suggestion
object NativeBuilderHelper {
```